descriptions/css/properties/line-height.json

```diff
@@ -0,0 +1,9 @@
+{
+  "css": {
+    "properties": {
+      "line-height": {
+        "__short_description": "The <strong><code>line-height</code></strong> <a href='https://developer.mozilla.org/docs/Web/CSS'>CSS</a> property sets the amount of space used for lines, such as in text."
```

Comment:
I just took the first sentence from the original page. This one is kinda fiendish, doing different things for different kinds of elements. Again, open to suggestions here, but it might help to look at the corresponding page.

Reply:
This is a weird case where the description is less helpful than the property name, since the name has "height" and the description is the less specific "amount of space".

Do you like:

The CSS line-height property sets the height of a line box. It's commonly used to set the distance between lines of text.

Reply:
I really like that. Thank you!